sourcegraph/about
# Guide/Vulnerabilities: Less is More

This closes #5245. It adds a new guide page and combines `WebinarLayout` -> `SimpleGatedPageLayout`

### Changelog

- Consolidates `WebinarLayout` to account for Webinar, Guides, and Report pages
- Adds `/guides/fixing-vulnerabilities-less-is-more.tsx`

### Note
- ~~PDF download on Form submission is not ready yet~~
- ~~There might be some content updates from Product, but the general layout should stay the same~~

### Test

1. Navigate to `/guides/fixing-vulnerabilities-less-is-more`
2. Please check that layout matches Figma specs
3. Ensure linting passes.
4. Ensure prettier has standardized the proposed changes.
5. Ensure dev and production builds work.
